test/ci-operator-configresolver-integration/run.sh

```diff
@@ -14,11 +14,20 @@ trap "rm -rf ${WORKDIR}" EXIT
 cd $WORKDIR
 # copy registry to tmpdir to allow tester to modify registry
 cp -a $ROOTDIR/test/ci-operator-configresolver-integration/ tests
-ci-operator-configresolver -config tests/configs -log-level debug &
+ci-operator-configresolver -config tests/configs -log-level debug -cycle 2m &
 PID=$!
 if ! timeout 10s bash -c 2>/dev/null -- "until diff tests/expected/openshift-installer-release-4.2.json <(curl 'http://127.0.0.1:8080/config?org=openshift&repo=installer&branch=release-4.2'); do sleep 1; done" &>/dev/null; then
     echo "diff: $(diff -ru tests/expected/openshift-installer-release-4.2.json <(curl 'http://127.0.0.1:8080/config?org=openshift&repo=installer&branch=release-4.2'))"
     kill $PID
+    wait $PID
+    exit 1
+fi
+cp tests/configs2/release-4.2/openshift-installer-release-4.2-golang111.yaml tests/configs/release-4.2/openshift-installer-release-4.2.yaml
+if ! timeout 10s bash -c 2>/dev/null -- "until diff tests/expected/openshift-installer-release-4.2-golang111.json <(curl 'http://127.0.0.1:8080/config?org=openshift&repo=installer&branch=release-4.2'); do sleep 1; done" &>/dev/null; then
```

Comment:
Suggestion: add a `/generation` (or whatever) endpoint that is incremented after each update and check it here. As it is, we'll block for up to 20s in case of a true positive.